core/new-gui/src/app/dashboard/component/feature-container/user-file-section/user-file-section.component.ts

```diff
@@ -87,8 +100,8 @@ export class UserFileSectionComponent {
     this.userFileService
       .downloadUserFile(userFileEntry.file)
       .pipe(untilDestroyed(this))
-      .subscribe(
-        (response: Blob) => {
+      .subscribe({
```

Comment:
can we revert those changes from this PR? since the TODO is not only to replace message with notification and it is irrelevant to this PR.

Reply:
just reverted the changes, will commit to the Texera/texera branch